pkg/test/mergeTemplates/21.08.0_lun_merge_21.08.0_extended.yaml

```diff
@@ -14,9 +14,6 @@ counters:
     - ^vserver => svm
 plugins:
   LabelAgent:
-    value_mapping:
-      - status state online `1`
-      - custom_status state online `1`
     value_to_num:
       - new_status state online online `0`
```

Comment:
should we add custom line 19 here also?

Reply:
good catch @Hardikl  - in this case, that's test code that is still correctly handled